asdf/tests/test_resolver.py

```diff
@@ -1,7 +1,7 @@
 import pytest
 
+from asdf._resolver import Resolver, ResolverChain
```

Comment:
Can this be moved to a local import down in `assert_extension_correctness`. Eventually, that method will need to be deprecated too. I would rather not expose the "new" private API in this module.